tool/ruby_vm/views/_leaf_helpers.erb

```diff
@@ -18,16 +18,16 @@ leafness_of_getglobal(VALUE gentry)
     else {
         /* We cannot write this function using a switch() because a
          * case label cannot be a function pointer. */
-        static rb_gvar_getter_t *const whitelist[] = {
+        static rb_gvar_getter_t *const allowlist[] = {
```

Comment:
`allowedList`?

Reply:
I found [this comment](https://english.stackexchange.com/questions/51088/alternative-term-to-blacklist-and-whitelist#comment95013_51104) and [here](https://github.com/rails/rails/issues/33677) suggest `allowlist`. What do you think?